frontend/src/components/VIconButton/VIconButton.vue

```diff
@@ -63,6 +64,13 @@ export default defineComponent({
       type: Object as PropType<ButtonProps>,
       default: () => ({ variant: "plain" }),
     },
+    /**
+     * the label for the button; This is used for accessibility purposes.
+     */
+    label: {
+      type: String,
```

Comment:
If we make the `label` prop accept translate results, we can drop the `.toString` from all the different places where it is passed.

```typescript
import { TranslateResult } from 'vue-i18n'
```

```suggestion
      type: [String, Object] as PropType<string | TranslateResult>,
```